.github/workflows/pr-test-lint.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
         uses: goreleaser/goreleaser-action@v6
         with:
           version: latest
-          args: release --snapshot --skip-publish --rm-dist
+          args: release --snapshot --skip=publish --clean
```

Comment:
is the skip=publish here correct? instead of skip-publish ?

Reply:
Yes:
![image](https://github.com/mondoohq/packer-plugin-cnspec/assets/827818/abe23362-8e7d-4f47-ad4d-97726591cc6d)